File: dices.js
function roll(dice_value) {
  var dices = [4, 6, 8, 10, 20, 100];

  if (dices.includes(dice_value)) {
    result = Math.floor(Math.random() * dice_value) + 1;
    value = "1d" + dice_value;
    return result;
  } else {
    console.log("Enter a valid number: (4, 6, 8, 10, 20, 100)");
  }
}

test = roll(20);

File: py/dices.py
import random


def roll(dice_value):
    '''
    Simulate roling dices by returning random numbers between 1 and 4, 6, 8, 10, 20, 100, other values are excluded.
    '''
    dices = [4, 6, 8, 10, 20, 100]

    if dice_value in dices:
        dice_result = random.randint(1, dice_value)
    else:
        print('Enter a valid number: (4, 6, 8, 10, 20, 100)')
    return dice_result


rand = roll(20)

print(f'1d{str(rand)}')
